services/EventTokenService.js

```diff
@@ -0,0 +1,68 @@
+const jwt = require("jsonwebtoken");
+const crypto = require("crypto");
+
+// A service class for Token generation and management
+
+class EventTokenService {
+  static #app_access_key = config.get("Event100ms.APP_ACCESS_KEY") || process.env.Event100ms.APP_ACCESS_KEY;
+  static #app_secret = config.get("Event100ms.APP_SECRET") || process.env.Event100ms.APP_SECRET;
+  #managementToken;
```

Comment:
I didnt get what is this for ?

Reply:
Hey @vvaibhavdesai,
Actually, this is an another PR which sets up the 100ms backend with the RDS backend. 
You can review it here: #972.

Reply:
what does this declaration mean ?
`#managementToken`

Reply:
private declaration which is used to store managementToken. Later in the constructor,  it is assigned a value using the getManagementToken method.